pkg/operator/utils/utils.go

```diff
@@ -72,6 +72,32 @@ func LoadInfrastructureTopology(c client.Client, logger log.FieldLogger) (config
 	return infra.Status.InfrastructureTopology, nil
 }
 
+// IsTimedTokenCluster answers a two part test to determine if we're running on a cluster enabled for timed access tokens (TAT)
+// like STS enabled cluster on AWS, GCP Workload Identity Federation or Azure AAD Pod Identity.
+//  1. Is credential mode Manual
+//  2. Is serviceAccountIssuer non-empty
+//
+// Both of these conditions must be true for any timed access token enabled clusters for the implementations mentioned above.
+func IsTimedTokenCluster(c client.Client, logger log.FieldLogger) (bool, error) {
+	credentialsMode, _, err := GetOperatorConfiguration(c, logger)
+	if err != nil {
+		logger.WithError(err).Error("error loading CCO configuration to determine mode")
+		return false, err
+	}
+	if credentialsMode != "Manual" {
```

Comment:
super-nit: is manual not a constant somewhere?

Reply:
yikes! investigating I find that mode referred to as both "manual" and "Manual" in yamls, docs and code.  Yes, there is a constant, that has it as "manual" (https://github.com/openshift/cloud-credential-operator/commit/1b384ccd0a8f587e2bf79281f241983ce9235ef5#diff-f6d93feef6d59ca8ced14b54fe0058d41f14de44450baf6b723cc1c2ec280c54R25) but it doesn't seemed to be used everywhere now....

So I'm a little leery to start using the constant version here until CCO team can review a bit more. I'd say it can wait until another day to get better?